### PR TITLE
Documentation addition

### PR DESCRIPTION
**Pull request by jbohren**
Basically emphasizes export/import functionality, and gives a sample yaml file.

### CODE REVIEW

**Comment by dirk-thomas** [comment]
Please start every sentence on a new line.

File: README.rst [hunk 1]
```diff
@@ -27,13 +27,33 @@ Optionally one path (or multiple paths) can be passed to search for repositories
...
+      url: https://github.com/vcstools/rosinstall/trunk
+      version: 748
+
+Or alternatively:
+
+.. code-block:: yaml
+
+  repositories:
+    vcstool: {type: git, url: "git@github.com:dirk-thomas/vcstool.git", version: master}
+    old_tools/rosinstall: {type: svn, url: "https://github.com/vcstools/rosinstall/trunk", version: 748}
```
Comment (dirk-thomas) [inline README.rst]: Why should the example be listed twice (only with a different yaml format)? vcstool should always write the more readable first format.

File: README.rst [hunk 2]
```diff
@@ -27,13 +27,33 @@ Optionally one path (or multiple paths) can be passed to search for repositories
...
   vcs status /path/to/several/repos /path/to/other/repos /path/to/single/repo
 
-Advanced features
------------------
+Exporting and importing sets of repositories
+--------------------------------------------
 
-Show log since last tag
-~~~~~~~~~~~~~~~~~~~~~~~
+Vcstool can export and import all the information required to reproduce the versions of a set of repositories. Vcstool uses a simple `YAML <http://www.yaml.org/>`_ format to encode this information. This format includes a root key ``repositories`` under which each local repository is described by a dictionary keyed by its relative path. Each of these dictionaries contains keys ``type``, ``url``, and ``version``.
```
Comment (dirk-thomas) [inline README.rst]: The `version` key is optional for some vcs (e.g. SVN).

Comment (jbohren) [inline README.rst]: That would be helpful to know!

File: README.rst (hunk 1 above)
Comment (jbohren) [inline README.rst]: I don't think it hurts to have it there.

**Comment by jbohren** [comment]
Added line breaks.

File: README.rst (hunk 1 above)
Comment (dirk-thomas) [inline README.rst]: Since it is redundant information and does not help a user in any way please remove the alternative yaml format.

**Comment by jbohren** [comment]
Alternate example removed.

**Comment by dirk-thomas** [comment]
Thank you for the documentation improvements. Can you please squash the commits before merging?

**Comment by jbohren** [comment]
> Thank you for the documentation improvements. Can you please squash the commits before merging?

Nope.

**Comment by dirk-thomas** [comment]
Squashed and merged: 22686951ad417a5dbaaad16ff9aa1a7d4d5ee6d2

**Pull request closed without merging**